src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+
+    // &:nth-child(-n + 1),
+    // &:nth-child(-n + 2),
+    // &:nth-child(-n + 3),
+    // &:nth-child(-n + 4),
+    // &:nth-child(-n + 5) {
+    //   background-image: url('../images/star-active.svg');
+    // }
```

Comment:
Remove the unnecessary empty lines to follow the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). Empty lines should be used sparingly to separate logical sections of code.